libs/utilities/src/lib/axios-error/axios-error.ts

```diff
@@ -0,0 +1,12 @@
+import { AxiosError } from 'axios';
+
+export const errorToAxiosError = (
+  error: unknown,
+  key = 'reason',
+): string | undefined => {
+  const { response } = error as AxiosError<{ [key: string]: string }>;
+
+  const errorMessage = response?.data[key];
+
+  return errorMessage;
```

Comment:
So the method name is `errorToAxiosError` which should just perform a cast. Let's keep this method doing what the name says.

```suggestion
  return error as AxiosError<{ [key: string]: string }>;
```

If you want to you can create a separate helper method called something like `axiosErrorToString = (error: unknown | AxiosError) => string`